kalitectl.py

```diff
@@ -480,12 +454,13 @@ def start(debug=False, daemonize=True, args=[], skip_job_scheduler=False, port=N
     if daemonize:
         
         from django.utils.daemonize import become_daemon
-        logfile = os.path.join(KALITE_HOME, "kalite.log") if os.environ.get("NAIVE_LOGGING", False) else None
-        if logfile:
-            print("With logging...")
-            become_daemon(out_log=logfile, err_log=logfile)
-        else:
-            become_daemon()
+        kwargs = {}
+        # Truncate the file
+        open(SERVER_LOG, "w").truncate()
+        print("Going to daemon mode, logging to {0}".format(SERVER_LOG))
+        kwargs['out_log'] = SERVER_LOG
+        kwargs['err_log'] = SERVER_LOG
```

Comment:
And yes, agreed, this would have happened both with the old and new approach anyways... I'm thinking that making failure output available is extremely crucial, so we really have to make it default. We need an option for the future to select a different file and switch it off.